sysdeps/managarm/include/abi-bits/xattr.h

```diff
@@ -0,0 +1 @@
+../../../../abis/mlibc/xattr.h
```

Comment:
Managarm wants to use the Linux ABI, so we should use the Linux ABI here.

Reply:
Save for support for interacting with linux-headers, it's identical